src/test/java/org/sonar/plugins/coverage/generic/ReportParserTest.java

```diff
@@ -58,13 +61,16 @@
   private SensorContext context;
   @Mock
   private ResourcePerspectives perspectives;
-  private File fileWithBranches;
-  private File fileWithoutBranch;
-  private File emptyFile;
+  private InputFile fileWithBranches;
+  private InputFile fileWithoutBranch;
+  private InputFile emptyFile;
+  private DefaultFileSystem fs;
 
   @Before
   public void before() {
     MockitoAnnotations.initMocks(this);
+    fs = new DefaultFileSystem();
+    fs.setBaseDir(new File("."));
```

Comment:
@mpaladin Highly subjective: I prefer to write `fs = new DefaultFileSystem().setBaseDir(baseDir);`